### PR TITLE
feat: Example of using DomParser with Offscreen API

### PR DESCRIPTION
**Pull request by 1natsu172**
@aklinker1 I’ve got a project just for fun, so I submit it since have the chance.

You just added a basic setup example to the repo, if you don't need this PR, please close.

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -283,14 +305,18 @@
     "browser.devtools.panels.create",
     "browser.devtools.panels.elements",
     "browser.downloads.download",
+    "browser.offscreen.closeDocument",
     "browser.offscreen.createDocument",
+    "browser.runtime)",
```
Comment (1natsu172) [inline metadata.json:310]: 🤔 Might need to trim: https://github.com/wxt-dev/examples/pull/5

Comment (aklinker1) [reply metadata.json:310]: I would rather update the code so the actual API you use inside `browser.runtime.xyz` shows up.

**Comment by aklinker1** [comment]
> You just added a basic setup example to the repo, if you don't need this PR, please close.

Nope, this is a great example, let's include it!

File: metadata.json (hunk 1 above)
Comment (1natsu172) [reply metadata.json:310]: Now that you mention it, you're right! 😄

**Review by aklinker1 — APPROVED** [review]
Made some small changes, checkout my commits if you're curious. Thanks for being the first contributor in this repo :)

**Comment by 1natsu172** [comment]
Thanks for some tweak codes asnd merged. 

Actually... the `)` syntax part was a copy and paste from [here](https://developer.chrome.com/docs/extensions/reference/api/offscreen#before_chrome_116_check_if_an_offscreen_document_is_open) lol. You're right, optional chaining is sufficient.

**Pull request merged**